bindata/assets/alerts/client-go-requests.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: client-go
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-client-go
+      rules:
+        # latency by:
+        # - job (kubelet, apiserver. controller-manager, ...)
+        # - destination host: internal-lb, kubernetes.default service, direct connection ...
+        - record: cluster:rest_client_request_duration_seconds:internal_load_balancer:histogram_quantile
```

Comment:
The following might be better suited since we want the scope of the recording rule before the metric name:

- cluster:internal_load_balancer:rest_client_request_duration_seconds:histogram_quantile
- internal_load_balancer:rest_client_request_duration_seconds:histogram_quantile

Reply:
I'm going with cluster:internal_load_balancer, since internal_load_balancer is an specific configuration of openshift clusters